packages/manager/src/features/Databases/DatabaseDetail/DatabaseResize/DatabaseResize.test.tsx

```diff
@@ -133,6 +133,251 @@ describe('database resize', () => {
+      const nodeRadioBtns = getByTestId('database-nodes');
+      expect(nodeRadioBtns.children.length).toBe(2);
+      expect(nodeRadioBtns).toHaveTextContent('$60/month $0.09/hr');
+      expect(nodeRadioBtns).toHaveTextContent('$140/month $0.21/hr');
+
+      const expectedCurrentSummary =
+        'Current Cluster: Nanode 1 GB $60/month 3 Nodes - HA $140/month';
+      const currentSummary = getByTestId('currentSummary');
+      expect(currentSummary).toHaveTextContent(expectedCurrentSummary);
+
+      const expectedResizeSummary =
+        'Resized Cluster: Please select a plan or set the number of nodes.';
+      const resizeSummary = getByTestId('resizeSummary');
+      expect(resizeSummary).toHaveTextContent(expectedResizeSummary);
+    });
+
+    it('should preselect cluster size in Set Number of Nodes', async () => {
+      const flags = {
+        dbaasV2: {
+          beta: false,
+          enabled: true,
+        },
+      };
+      const mockDatabase = databaseFactory.build({
+        cluster_size: 3,
+        type: 'g6-nanode-1',
+      });
+      const { getByTestId } = renderWithTheme(
+        <DatabaseResize database={mockDatabase} />,
+        { flags }
+      );
+      await waitForElementToBeRemoved(getByTestId(loadingTestId));
+      const selectedNodeRadioButton = getByTestId(
+        `database-node-${mockDatabase.cluster_size}`
+      ).children[0].children[0] as HTMLInputElement;
+      expect(selectedNodeRadioButton).toBeChecked();
+    });
+
+    it('should disable visible lower node selections', async () => {
+      const flags = {
+        dbaasV2: {
+          beta: false,
+          enabled: true,
+        },
+      };
+      const mockDatabase = databaseFactory.build({
+        cluster_size: 3,
+        type: 'g6-nanode-1',
+      });
+      const { getByTestId } = renderWithTheme(
+        <DatabaseResize database={mockDatabase} />,
+        { flags }
+      );
+      await waitForElementToBeRemoved(getByTestId(loadingTestId));
+      const selectedNodeRadioButton = getByTestId('database-node-1').children[0]
+        .children[0] as HTMLInputElement;
+      expect(selectedNodeRadioButton).toBeDisabled();
+    });
+
+    it('should set price, enable resize button, and update resize summary when a new number of nodes is selected', async () => {
+      const flags = {
+        dbaasV2: {
+          beta: false,
+          enabled: true,
+        },
+      };
+      const mockDatabase = databaseFactory.build({
+        cluster_size: 1,
+        type: 'g6-nanode-1',
+      });
+      const { getByTestId, getByText } = renderWithTheme(
+        <DatabaseResize database={mockDatabase} />,
+        { flags }
+      );
+      await waitForElementToBeRemoved(getByTestId(loadingTestId));
+      // Mock clicking 3 Nodes option
+      const selectedNodeRadioButton = getByTestId('database-node-3').children[0]
+        .children[0] as HTMLInputElement;
+      fireEvent.click(selectedNodeRadioButton);
```

Comment:
we should prefer `userEvent` over `fireEvent` whenever possible: https://testing-library.com/docs/user-event/intro/#differences-from-fireevent

```suggestion
      await userEvent.click(selectedNodeRadioButton);
```

Reply:
Thanks @bnussman-akamai! I've included this changes in the test file!